app/views/admin/team_members/_form.html.haml

```diff
@@ -8,12 +8,21 @@
     %fieldset
       %legend Basics
       = f.input :name, required: true
+      = f.input :title
+      %p
+        Leadership
+        %br
+        = f.check_box :is_leader
```

Comment:
If you don't mind too much, let's stick with the existing (imperfect) formatting here. The spacing may be off, or less than desirable UI, but we will want to change that with global styling of the admin form interfaces, having one-offs like this could jeopardize that.